sphinx_selective_exclude/eager_only.py

```diff
@@ -22,6 +22,7 @@
 #
 import sphinx
 from docutils.parsers.rst import directives
+from sphinx.util import logging
 
```

Comment:
```suggestion


logger = sphinx.util.logging.getLogger(__name__)

```
Addresses https://github.com/pfalcon/sphinx_selective_exclude/pull/7#discussion_r334138615